lms/djangoapps/support/static/support/jsx/entitlements/components/Main/Main.jsx

```diff
@@ -7,16 +7,23 @@ import EntitlementSupportTableContainer from '../Table/EntitlementSupportTableCo
 import EntitlementFormContainer from '../EntitlementForm/container.jsx';
 
 const Main = props => (
-  <div>
+  <div className="contact-us-wrapper entitlement-support-wrapper">
     <StatusAlert
       alertType="danger"
       dialog={props.errorMessage}
       onClose={props.dismissErrorMessage}
       open={!!props.errorMessage}
     />
-    <h2>
-      Student Support: Entitlement
-    </h2>
+    <div className="create-new">
```

Comment:
It seems a little strange to me to place the header inside of a div with className 'create-new' since they aren't really related so much.

Reply:
Yeah, that was something I was a little bit if-y on as well. Visually I do want them to be lined up, so I wanted them to be in a flexbox together, but it definitely could be named differently. Unless you think I should altogether pursue a different approach to achieve this visual result